TUTORIAL.adoc

```diff
@@ -0,0 +1,537 @@
+= Meander Tutorial
+:external-libs: meander[https://raw.githubusercontent.com/no-prompt/meander/delta/src]
+:preamble: (ns meander.examples (:require [meander.delta.strategy :as m]))
```

Comment:
The namespace should be `meander.strategy.delta`

Reply:
Good catch.